configure.ac

```diff
@@ -157,11 +157,11 @@ if test "x$enable_debug" = xyes; then
     if test "x$GCC" = xyes; then
         CFLAGS="$CFLAGS -g3 -O0"
     fi
-    
+
```

Comment:
as discussed on IRC, I'd prefer to not have the unnecessary whitespace changes here in sections you're otherwise not touching